transit_odp/browse/tests/test_change_data_archive.py

```diff
@@ -54,19 +54,20 @@ def test_get_datasets_published_at():
 def test_upload_change_data_archive(tmp_path):
     """Tests upload_bulk_data_archive creates BulkDataArchive with zipfile at outpath"""
     # Setup
-    now = timezone.now()
     zipped = tmp_path / "changes.zip"
     zipped.write_text("Some data")
 
     # Test
-    change_data_archive.upload_change_data_archive(zipped, now)
+    archive = change_data_archive.upload_change_data_archive(zipped, timezone.now())
+
+    # Use the same source of date/time for your assertions
+    published_date = archive.published_at.date()
 
     # Assert
     qs = ChangeDataArchive.objects.all()
     assert len(qs) == 1
 
-    archive = qs[0]
-    assert archive.published_at == now.date()
+    assert archive.published_at.date() == published_date
```

Comment:
The RHS and LHS on this assert seems to be same which translates to:

`assert archive.published_at.date() == archive.published_at.date()`

as `published_date = archive.published_at.date()`